### DISCUSSION TITLE
Welcome to did Discussions!

### DISCUSSION THREAD

**Discussion by bcallaway11** [bcallaway11/did#109] [Announcements]
Hi all,

I am opening up "Discussions" for the `did` package.  My aims are:

* Move some existing *common* issues into discussions, so that they are easier for other users to find
* Have a place for some ongoing discussions about directions for new features/improvements

Brant

**Comment by shaodiana** [comment]
webpage (https://bcallaway11.github.io/did/articles/did-basics.html) seems not updated/in-sync with the source

I recently tried following along the R codes on the "Getting Started with the did Package" page, and it seems that it's not in sync with the source (https://github.com/bcallaway11/did/blob/HEAD/vignettes/did-basics.Rmd). For example, on the "getting started" page under "Building the dataset" section, "sp" was not defined but on the source page, it is defined as the following. 
sp <- reset.sim()
sp$te <- 0 

This issue also occurs for the Pre-testing page (https://bcallaway11.github.io/did/articles/pre-testing.html) where sp was not defined first and therefore dta cannot be generated with build_sim_dataset(). 

Thank you.
